src/lib/tests/Generator.class.test.ts

```diff
@@ -107,7 +107,7 @@ describe('Generator Class', () => {
               await pipelineParallelGenerators.run()
               const file = fs.readFileSync(filePath, {encoding: 'utf-8'})
               const fileLines = file.split('\n').sort()
-              expect(fileLines.length).to.equal(873)
+              expect(fileLines.length).to.equal(741)
```

Comment:
This is lower because of the duplicate triples that were being generated.

Reply:
I can't seem to find where the duplicatie triples issue is solved. Apparently you did (great!) but can you link me to the line(s)? I just want to make sure we do not introduce a OoM issue.